src/config/tokens/themes/hacker/keys/_hacker-theme-keys.scss

```diff
@@ -53,6 +53,11 @@ $theme-hacker-warning-color-hover: $color-desert;
 $theme-hacker-warning-color-selected: $color-brass;
 $theme-hacker-warning-color-selected-hover: $color-aztec_gold;
 
+// brand colors
+$theme-hacker-brand-color: $color-burnt-orange;
```

Comment:
is this correct?

Reply:
What's wrong here?
Hacker colors are just equal to the ones we have in black theme - so seems fine

Reply:
Just making sure it is the correct mapping. I see the mapping to some pink color
![image](https://github.com/mondaycom/monday-ui-style/assets/6093192/d270baf3-5dcb-4cc0-81ab-8474411f0b48)